src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/MovementResourceTest.kt

```diff
@@ -343,8 +305,6 @@ class MovementResourceTest : ResourceTest() {
     fun `should return 404 when does not have agency in caseload`() {
       webTestClient.get().uri("/api/movements/rollcount/SFI/in-reception")
         .headers(setAuthorisation(listOf("")))
-        .header("Content-Type", MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Glad you removed these - I think there are a few like this that I accidentally added in